frontend/app/views/fragments/form/featureChoicePanelPayPal.scala.html

```diff
@@ -0,0 +1,49 @@
+@(tier: com.gu.salesforce.Tier)(implicit request: RequestHeader)
+
+@import com.gu.salesforce.Tier.{Partner, Patron}
+@import model.FeatureChoice
+@import model.FreeEventTickets
+@import configuration.Config
+
+@partnerChoice() = {
+    <section class="form-panel">
+        <div class="form-panel__heading">
+            <h2 class="form-panel__headline">Select benefit option</h2>
+            <div class="form-panel__edit js-edit-benefits is-hidden">edit</div>
+        </div>
+        <div class="form-panel__content js-form-panel-benefits">
+            <fieldset class="fieldset js-feature-choice-container">
```

Comment:
Good spot! I had a look through the git history, and it seems this was only ever added to the templates, it never existed in the JavaScript for some reason...